docker/gcp_pudl_etl.sh

```diff
@@ -65,6 +65,12 @@ function run_pudl_etl() {
     && touch "$PUDL_OUTPUT/success"
 }
 
+function write_pudl_datapackage() {
+    echo "Writing PUDL datapackage."
+    python -c "from pudl.metadata.classes import PUDL_PACKAGE; print(PUDL_PACKAGE.to_frictionless().to_json())" > "$PUDL_OUTPUT/pudl_datapackage.json"
```

Comment:
Is this the right place for the `pudl_datapackage.json` to go, given that the parquet files are output under `$PUDL_OUTPUT/parquet/`? The resource paths in the descriptor refer to the current directory, suggesting that the parquet files and the datapackage.json file should be in the same directory.  Or is this location anticipating that the parquet files will get moved into `$PUDL_OUTPUT` eventually? Or that wherever the JSON is deployed to will be alongside all the parquet files? For more ongoing consistency might we want to output the datapackage.json alongside the parquet files and then also move them together when they move?

Should we make the name parquet specific, given that we could also have an SQLite or DuckDB database that's annotated by a datapackage.json file?

Reply:
You're right on all counts. I put this in `$PUDL_OUTPUT` because that's where the parquet files end up in distribution, and it does make more sense to call it `pudl_parquet_datapackage.json` and put it in `parquet/`. Easy fixes.